README.md

```diff
@@ -105,3 +105,35 @@ When you choose a name other than "editorconfig.json", enter the path of the edi
 ***
 
 After the above work, restart Firefox.
+
+***
+
+## Troubleshooting
+
+If something goes wrong, check the browser console (Ctrl + Shift + J).
+
+```
+Error: Attempt to postMessage on disconnected port
+```
+
+* Windows: Is the registory saved correctly?
+* Linux / Mac: Is "withexeditorhost.json" saved in the right location?
+* When you start Firefox, is Node.js process executed too?
+  * If not, make sure there is $PATH environment variable for Node.js.
```

Comment:
"If not, make sure that Node.js's installation directory is listed in the `$PATH` environment variable."